src/builder/Actions.sol

```diff
@@ -245,6 +256,11 @@ library Actions {
         address paymentToken;
         string paymentTokenSymbol;
         uint256 paymentMaxCost;
+        // The secret used to generate the hash chain for a replayable operation. For non-replayable
```

Comment:
Should we return `nonce` and `nonceSecret`? Generally we've been on "verify" mode on the client (as opposed to "generate"). That is, we run a check that `hash_n(input.nonceSecret, input.totalPlays) == input.nonce`

Reply:
The actual `nonce` is on the `QuarkOperation`, so I don't think we need to put it here.